src/rust/ensogl/src/animation/easing.rs

```diff
@@ -177,13 +178,22 @@ pub struct AnimatorData<T,F,Cb> {
 }
 
 impl<T:Value,F,Cb> AnimatorData<T,F,Cb>
-    where F  : AnyFnEasing,
-          Cb : Callback<T> {
+where F:AnyFnEasing, Cb:Callback<T> {
+    fn new(start:T, end:T, tween_fn:F, callback:Cb) -> Self {
+        let duration     = Cell::new(1000.0);
```

Comment:
Literal, from this code I don't really know what the units are. (though that's more of an issue with the `duration` field type)

Reply:
this is yet to be cleaned, its in ms